### ISSUE TITLE
Error extracting player bios

### ISSUE THREAD

**Issue by pacoca43** [JaseZiv/worldfootballR#348]
I'm having a trouble trying to extract player bios.
The Code:
```
hazard_bio <- tm_player_bio(player_url = "https://www.transfermarkt.com/eden-hazard/profil/spieler/50202")
dplyr::glimpse(hazard_bio)
```

The error:
**Error in `dplyr::mutate()`:
ℹ In argument: `date_of_birth = .tm_fix_dates(.data[["date_of_birth"]])`.
Caused by error in `.data[["date_of_birth"]]`:
! Column `date_of_birth` not found in `.data`.
Run `rlang::last_trace()` to see where the error occurred.**

Thanks a lot.

**Comment by MioMio99** [comment]
same here

**Comment by JaseZiv** [comment]
@pacoca43 @MioMio99

Can you please confirm the version of `worldfootballR` that you are on? 

I patched a fix a few weeks ago. Make sure you install the latest version from GitHub.

**Comment by pacoca43** [comment]
My version of worldfootballR  is 0.6.4.0012. Thanks

**Comment by JaseZiv** [comment]
This was fixed in the latest update (0.6.4.0014).

Install the latest version from GitHub and that should solve your problems.

**Comment by pacoca43** [comment]
Thanks for your help and sorry.
